src/domain/services/AIService.ts

```diff
@@ -371,16 +369,29 @@ export class AIService {
       );
     }
 
-    const ais = await prismadb.aI.findMany({
-      select: {
-        ...listAIResponseSelect(),
-        chats: {
-          where: {
-            userId,
-            isDeleted: false,
-          },
+    let categoryType : CategoryType | undefined;
+    try {
+      // @ts-ignore
+      categoryType = CategoryTypesHardcoded[categoryId] as CategoryType
+    } catch (error) {
+      console.error("Error parsing category type", error);
+      categoryType = undefined;
+    }
+
+    let selectClause = {...listAIResponseSelect(), chats: {
```

Comment:
Why do we need to include chats in the select?

Reply:
Not Sure, Should I remove it. It is already there.